### DISCUSSION TITLE
Noob Question - How do I create/delete directories?

### DISCUSSION THREAD

**Discussion by CliffordSpainhour** [focus-editor/focus#390] [General]
I'm on windows. 

I see that we can create new files and save them. 

I find myself also wanting to create directories, sometimes at the same time as saving a file.

Also sometimes I mistype/spell a file and want to delete it. 

Do we handle this with commands or is there something I'm missing? Beyond the scope of this editor?

**Comment by focus-editor** [comment]
This is sadly not yet implemented, but we'll get there eventually.
